server/app.py

```diff
@@ -21,22 +21,36 @@
 jwt = JWTManager(app)
 
 
-from models import UserModel, ProspectModel, TagModel, CampaignModel, StepModel, EmailTemplateModel
-from resources import UserResources, GmailResources, CampaignResources, ProspectsResources, TemplateResources
+from models import (
+    UserModel, 
+    ProspectModel,
+    TagModel, 
+    CampaignModel, 
+    StepModel, 
+    EmailTemplateModel
+)
+from resources import (
+    UserResources, 
+    GmailResources, 
+    CampaignResources, 
+    ProspectsResources,
+    TemplateResources
+)
 
 
 
 api.add_resource(UserResources.UserRegister, '/register')
 api.add_resource(UserResources.UserLogin, '/login')
-api.add_resource(UserResources.UserCampaigns, '/campaigns')
 api.add_resource(UserResources.UserProspects, '/prospects')
+api.add_resource(TemplateResources.AllTemplates, '/templates/all')
+api.add_resource(TemplateResources.OneTemplate, '/templates')
```

Comment:
We should probably make this more RESTful, I would change the API to:
GET /templates -> gets all templates
POST /templates -> creates one template 
PUT /templates/:id -> edit a template

Reply:
Updated those routes